example/gluon/lipnet/README.md

```diff
@@ -0,0 +1,110 @@
+# LipNet: End-to-End Sentence-level Lipreading
+
+---
+
+Gluon inplementation of [LipNet: End-to-End Sentence-level Lipreading](https://arxiv.org/abs/1611.01599)
+
+![net_structure](asset/network_structure.png)
+
+## Requirements
+- Python 3.6.4
+- MXnet 1.3.0
+
+
+## Test Environment
+- 4 CPU cores
+- 1 GPU (Tesla K80 12GB)
+
+
+## The Data
+- The GRID audiovisual sentence corpus (http://spandh.dcs.shef.ac.uk/gridcorpus/)
+- Video: (normal)(480 M each)
+- Align: word alignments(190 K each) 
+
+## Prepare the Data
+### Download the data
+- arguments
+  - src_path : Path for videos (default='./data/mp4s/')
+  - align_path : Path for aligns (default='./data/align/')
+  - n_process : num of process (default=1)
+
+```
+cd ./utils && python download_data.py
```

Comment:
I changed it to resume the download process.
(1) It check if the folder such as 's1' exists (2) If there is the folder, it skips. Otherwise,  it download the file and unzip the file.